content/en/docs/IV/modules/hosp/_index.md

```diff
@@ -1,11 +1,11 @@
 ---
 title: "Hosp"
-linkTitle: "hosp"
+linkTitle: "Hosp"
 date: 2020-08-10
 weight: 20
 description: >
-  The Hosp module provides all data acquired from the hospital wide electronic health record. Information covered includes laboratory measurements, microbiology, medication administration, and billed diagnoses.
+  The Hosp module provides all data acquired from the hospital wide electronic health record. Information covered includes patients and admissions information, laboratory measurements, microbiology, medication administration, and billed diagnoses.
```

Comment:
Suggestion: "Information covered includes patient and admission information, laboratory measurements, microbiology, medication administration, and billed diagnoses."

Reply:
Fixed